apps/binding.go

```diff
@@ -65,7 +65,7 @@ package apps
 //              "bindings": [
 //                  {
 //                      "location": "send-button",
-//                      "icon": "http://localhost:8080/static/icon.png",
```

Comment:
Should we create "broken examples" for the sake of testing?

Reply:
Good point :+1: